docs/source/build.md

```diff
@@ -220,27 +226,56 @@ The Python packages can also be uninstalled using the `build.sh` script:
 ./build.sh pylibraft raft-dask --uninstall
 ```
 
-### Documentation
+### Using Cmake directly
```

Comment:
Let's replace all "Cmake" or "cmake" with "CMake" unless referring to the executable `cmake`. Just want to avoid leaving a comment for every instance to aid with readability of the feedback. :)
```suggestion
### Using CMake directly
```